deploy/examples/images.txt

```diff
@@ -8,4 +8,4 @@
  registry.k8s.io/sig-storage/csi-provisioner:v4.0.1
  registry.k8s.io/sig-storage/csi-resizer:v1.10.1
  registry.k8s.io/sig-storage/csi-snapshotter:v7.0.2
- rook/ceph:master
+ docker.io/rook/ceph:master
```

Comment:
With this change, the sort order of the images changed, looks like the CI expects it at the top of the file now